src/chronometer.js

```diff
@@ -1,33 +1,47 @@
 class Chronometer {
   constructor() {
-    // ... your code goes here
-  }
+  this.currentTime = 0;
+  this.intervalId = null;
+  }   
+  
 
   start(printTimeCallback) {
-    // ... your code goes here
-  }
-
+    this.intervalId = setInterval(() => {
+      console.log(this.currentTime);
+      this.currentTime++;
+      if(printTimeCallback){
+        printTimeCallback()
+      }
+    }, 1000);
+  };
   getMinutes() {
-    // ... your code goes here
-  }
-
+    return Math.floor(this.currentTime/60);
+   }
   getSeconds() {
-    // ... your code goes here
+     return Math.floor(this.currentTime % 60);
   }
-
   computeTwoDigitNumber(value) {
-    // ... your code goes here
+    if(value < 10){
+  return `0${value}`
+  } else {
+    return `${value}`;
   }
-
+}
   stop() {
-    // ... your code goes here
+    clearInterval(this.intervalId)
   }
-
   reset() {
-    // ... your code goes here
-  }
-
-  split() {
-    // ... your code goes here
+     return this.currentTime = 0;
+     }
+  
+  split() { 
+    if(this.computeTwoDigitNumber){
```

Comment:
This if statement is unnecessary. Good job on the return, and there's another way to do it, by creating variables, like:

```
let minutes = this.computeTwoDigitNumber(this.getMinutes());
let seconds = this.computeTwoDigitNumber(this.getSeconds());
return `${minutes}:${seconds}`;
```